packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -2,198 +2,436 @@
+  }
+
+  /// Attempts a fling gesture starting at [startLocation], moving by [offset]
+  /// with the given [velocity].
+  ///
+  /// See [ElementTreeTester.flingFrom] for details.
+  void flingFrom(Point startLocation, Offset offset, double velocity, { int pointer: 1 }) {
+    elementTreeTester.flingFrom(startLocation, offset, velocity, pointer: pointer);
+  }
+
+  /// Begins a gesture at a particular point, and returns the
+  /// [TestGesture] object which you can use to continue the gesture.
+  TestGesture startGesture(Point downLocation, { int pointer: 1 }) {
+    return elementTreeTester.startGesture(downLocation, pointer: pointer);
+  }
+
+  /// Returns the size of the element corresponding to the widget located by
+  /// [finder].
+  ///
+  /// This is only valid once the element's render object has been laid out at
+  /// least once.
+  Size getSize(Finder finder) {
+    assert(finder != null);
+    Element element = finder.findFirst(this);
+    return elementTreeTester.getSize(element);
+  }
+
+  /// Returns the point at the center of the widget found by [finder].
+  Point getCenter(Finder finder) {
+    Element element = finder.findFirst(this);
+    return elementTreeTester.getCenter(element);
+  }
+
+  /// Returns the point at the top left of the given element.
+  Point getTopLeft(Finder finder) {
+    Element element = finder.findFirst(this);
+    return elementTreeTester.getTopLeft(element);
+  }
+
+  /// Returns the point at the top right of the given element. This
+  /// point is not inside the object's hit test area.
+  Point getTopRight(Finder finder) {
+    Element element = finder.findFirst(this);
+    return elementTreeTester.getTopRight(element);
+  }
+
+  /// Returns the point at the bottom left of the given element. This
+  /// point is not inside the object's hit test area.
+  Point getBottomLeft(Finder finder) {
+    Element element = finder.findFirst(this);
+    return elementTreeTester.getBottomLeft(element);
+  }
+
+  /// Returns the point at the bottom right of the given element. This
+  /// point is not inside the object's hit test area.
+  Point getBottomRight(Finder finder) {
+    Element element = finder.findFirst(this);
+    return elementTreeTester.getBottomRight(element);
   }
-  dynamic _pendingException;
 }
 
-void testWidgets(callback(WidgetTester tester)) {
-  new FakeAsync().run((FakeAsync async) {
-    FlutterExceptionHandler oldHandler = FlutterError.onError;
-    try {
-      WidgetTester tester = new WidgetTester._(async);
-      FlutterError.onError = (FlutterErrorDetails details) {
-        if (tester._pendingException != null) {
-          FlutterError.dumpErrorToConsole(tester._pendingException);
-          FlutterError.dumpErrorToConsole(details.exception);
-          tester._pendingException = 'An uncaught exception was thrown.';
-          throw details.exception;
+/// Provides lightweight syntax for getting frequently used widget [Finder]s.
+class CommonFinders {
+  const CommonFinders();
```

Comment:
It would prevent developers from extending it to support their own custom finders. I might, for example, want a utility `find.zippyUnder(anotherFinder)`.